packages/extension-chakra-storefront/src/components/header/index.jsx

```diff
@@ -326,7 +326,13 @@ const Header = ({
                             {...styles.icons}
                             variant="unstyled"
                             onClick={() => {
-                                openModal()
+                                if (
+                                    !window.location.pathname.includes(
+                                        storeLocatorExtension.config.path
+                                    )
+                                ) {
+                                    openModal()
+                                }
```

Comment:
I'm not a fan of this... but i also don't know if there is a better way to do it.